src/RunTests/README.md

```diff
@@ -0,0 +1,30 @@
+# Microsoft.Build.RunTests
+
+The `Microsoft.Build.RunTests` MSBuild SDK adds support for running tests from MSBuild, similarly to how one would use `dotnet test`.
+
+## Usage in `Directory.Packages.Props`
+In your `Directory.Packages.props`:
+```xml
+<Project>
+  <ItemGroup>
+    <!-- <PackageVersion> elements here -->
+  </ItemGroup>
+  <ItemGroup>
+    <GlobalPackageReference Include="Microsoft.Build.RunTests" Version="1.0.0" />
+  </ItemGroup>
+</Project>
+```
+This example will include the `Microsoft.Build.RunTests` task for all NuGet-based projects in your repo.
+
+## Example
+To run tests
+```
+$env:MSBUILDENSURESTDOUTFORTASKPROCESSES=1
+msbuild /nodereuse:false /t:MSBuildRunTests
```

Comment:
Is `MSBuildRunTests` the final name that user is expected to use, or is this a placeholder. Such long name steals from the appeal for me, because when someone wanted to run the target directly they would use `VSTest` now.

Reply:
The name could be something shorter. I am open to any suggestions

Reply:
I think `VSTest` is best as it leaves the door open for different test platform we would want to provide.

Reply:
Updated